interface/globals.php

```diff
@@ -12,6 +12,12 @@
     die($response);
 }
 
+// Logic to prevent any session init's while a session is active. PHP 7.2
+// PHP_SESSION_ACTIVE ='s 2
+//
+session_start();
```

Comment:
I highly suspect this will break session stuff in OpenEMR. For example, if view the source of main openemr after log in, will see following at top:
``````
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html>
<head>
<title>OpenEMR</title>

<script type="text/javascript">
// login.php makes sure the session ID captured here is different for each
// new login.  We maintain it here because most browsers do not have separate
// cookie storage for different top-level windows.  This function should be
// called just prior to invoking any server script that requires correct
// session data.  onclick="top.restoreSession()" usually does the job.
//
var oemr_session_name = 'OpenEMR';
var oemr_session_id   = 'ciakip11jfmqhciedot87o25j7';
var oemr_dialog_close_msg = 'OK to close this other popup window?';
``````
however, with this code, see:
``````
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html>
<head>
<title>OpenEMR</title>

<script type="text/javascript">
// login.php makes sure the session ID captured here is different for each
// new login.  We maintain it here because most browsers do not have separate
// cookie storage for different top-level windows.  This function should be
// called just prior to invoking any server script that requires correct
// session data.  onclick="top.restoreSession()" usually does the job.
//
var oemr_session_name = 'PHPSESSID';
var oemr_session_id   = 'me7jjo2imei1gsa7q68v22vnbv';
var oemr_dialog_close_msg = 'OK to close this other popup window?';
``````
Note that no longer setting the session name to OpenEMR as should be doing below.